README.md

```diff
@@ -24,24 +32,135 @@ The optimal choice and combination of bridging species may vary from one genomic
+## Quick start
+Project any regions of interest as a BED-file provided:
+1. `regions.bed`
+2. `pwaln file generated for the species pair`
 
-Alternatively, we provide a pipeline to compute your own alignment collections for your choice of species.
-For that, run `compute_alignments/compute_pairwise_alignments`
-A Snakefile guides through the whole alignment process from fasta to chain file.
-With that, you can create the final collection of pairwise alignments between reference, target and a given set of bridging species that is used for running IPP.
+For example, to project a set of enhancers from mm39 (reference) to galGal6 (target) using 10 cores:
 
+```bash
+python project.py -o ipp_output/ -n 10 ./enhancers.bed mm39 galGal6 ./mm39.galGal6.pwaln.bin
+```
 
 ## Usage
+Run `python project.py -h` for a detailed description:
+
+```
+positional arguments:
+  regions_file          Bed file containing genomic coordinates. regions with width > 1 will be centered.
+  ref                   Reference species
+  qry                   Query species
+  path_pwaln
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -o OUT_DIR, --out_dir OUT_DIR
+                        Directory for output files (default: ./ipp-data/)
+  -sdc SCORE_DC, --score_DC SCORE_DC
+                        Score threshold for direct conservation detection (default: 0.98)
+  -sic SCORE_IC, --score_IC SCORE_IC
+                        Score threshold for indirect conservation detection (default: 0.84)
+  -ddc DISTANCE_DC, --distance_DC DISTANCE_DC
+                        Distance threshold for direct conservation detection. Regions up to this distance to the closest anchor will be considered as directly conserved. Used instead of score_threshold.
+                        (default: None)
+  -dic DISTANCE_IC, --distance_IC DISTANCE_IC
+                        Distance threshold for indirect conservation detection. Regions up to this distance to the closest anchor will be considered as indirectly conserved. Used instead of score_threshold.
+                        (default: None)
+  -dfc DISTANCE_FC, --distance_FC DISTANCE_FC
+                        Distance threshold for functional conservation detection. Regions up to this distance to the closest region in the target_bedfile will be considered as functionally conserved.
+                        (default: 500)
+  -n N_CORES, --n_cores N_CORES
+                        Number of CPUs (default: 1)
+  -t TARGET_BEDFILE, --target_bedfile TARGET_BEDFILE
+                        Functional regions in target species to check for overlap with projections for classification (default: None)
+  -q, --quiet           Do not produce any log output (default: False)
+  -v, --verbose         Produce additional debugging output (default: False)
+  -c, --simple_coords   Make coord numbers in debug output as small as possible (default: False)
+  -a, --include_anchors Include anchors in results table (default: False)
+```
```

Comment:
```suggestion
Run `python project.py -h` for a detailed description of the arguments.
```
Remove the detailed description, it's a duplicate of the shell output. It's sufficient to point out how to retrieve it.